config/run-axe.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const fs = require('fs');
+const urlParse = require('url').parse;
+const chromeLauncher = require('chrome-launcher');
+const CDP = require('chrome-remote-interface');
+const chalk = require('chalk');
+const runServer = require('./static-server');
+
+const REMOTE_CHROME_URL = process.env['REMOTE_CHROME_URL'];
+const AXE_JS = fs.readFileSync(__dirname + '/../node_modules/axe-core/axe.js');
+const PAGES = [
+  '/',
+  '/page-templates/landing/',
+  '/page-templates/docs/',
+];
+
+function launchChromeLocally(headless=true) {
+  return chromeLauncher.launch({
+    chromeFlags: [
+      '--window-size=412,732',
+      '--disable-gpu',
+      headless ? '--headless' : ''
+    ]
+  });
+}
+
+function getRemoteChrome() {
+  const info = urlParse(REMOTE_CHROME_URL);
+  if (info.protocol !== 'http:')
+    throw new Error(`Unsupported protocol: ${info.protocol}`);
+  return new Promise(resolve => {
+    resolve({
+      host: info.hostname,
+      port: info.port,
+      kill() { return Promise.resolve(); }
+    });
+  });
+}
+
+// This function is only here so it can be easily .toString()'d
+// and run in the context of a web page by Chrome. It will not
+// be run in the node context.
+function runAxe() {
+  return new Promise((resolve, reject) => {
+    window.axe.run((err, results) => {
+      if (err) return reject(err);
+      resolve(JSON.stringify(results.violations));
+    });
+  });
+}
+
+let getChrome = REMOTE_CHROME_URL ? getRemoteChrome : launchChromeLocally;
+
+Promise.all([runServer(), getChrome()]).then(([server, chrome]) => {
+  const chromeHost = chrome.host || 'localhost';
+  console.log(`Static file server is listening at ${server.url}.`);
+  console.log(`Chrome is debuggable on http://${chromeHost}:${chrome.port}.`);
+  console.log(`Running aXe on:`);
+
+  CDP({
+    host: chrome.host,
```

Comment:
Oh, it will just be `undefined` if it doesn't exit, which CDP just then defaults to localhost--but I agree, I think it'd be less confusing / more explicit if we just passed `chromeHost` here.

I was hoping that the API for `chrome` would set `host` to whatever the ultimate "resolved" value was, whether that was `localhost` (in the case where Chrome is launched locally) or something explicit (in the case where we connect to a remote Chrome in e.g. a different Docker container).  However, that wasn't the case--in the case where Chrome is launched locally, `host` isn't set at all, so I had to manually default it to `localhost` via that `chromeHost` variable. It's a bit annoying because the APIs that connect all these different Chrome packages isn't quite as consistent as I'd hoped, and this is a workaround for that.